README.md

```diff
@@ -179,61 +153,55 @@ Armbian [releases](https://docs.armbian.com/Release_Changelog/) quarterly at the
     └── u-boot                           User: universal boot loader patches
 ```
 
-<p align=right><a href=#table-of-contents>⇧</a></p>
-
-## Support
-
-- Have you found a bug in the **build framework**? 
-
-    Try to recreate it with a clean build framework clone. Then search for [existing and closed issues](https://github.com/armbian/build/issues). If you don't find it there, [open a new issue](https://github.com/armbian/build/issues/new).
-
-- Do you have troubles **elsewhere**? 
+## Contribution
 
-    Armbian is free software and provides **best effort help** through [community forums](https://forum.armbian.com/). If you can't find answer there and/or with help of [general project search engine](https://www.armbian.com/search) and [documentation](https://docs.armbian.com), consider [hiring an expert](https://www.debian.org/consultants/).
-
-- Personalised support?
+- **You don't need to be a programmer to help!**
```

Comment:
I would remove bold and add a link to https://forum.armbian.com/staffapplications/